src/commands/commit/withClient/index.js

```diff
@@ -25,10 +25,20 @@ const withClient = async (answers: Answers): Promise<void> => {
 
     if (configurationVault.getAutoAdd()) await execa('git', ['add', '.'])
 
-    await execa('git', ['commit', '-m', title, '-m', answers.message], {
-      buffer: false,
-      stdio: 'inherit'
-    })
+    await execa(
+      'git',
+      [
+        'commit',
+        configurationVault.getAutoAdd() ? '-am' : '-m',
```

Comment:
`-am` here technically is not needed since we're running `git add .` if the option is enabled but I guess it's a way of improving the error handling, can we extract this into a variable?

```suggestion
        configurationVault.getAutoAdd() ? '-am' : '-m',
```

Reply:
https://github.com/carloscuesta/gitmoji-cli/blob/5651277b98e949b64d4dc643afe610cfa1cbc929/src/commands/commit/withClient/index.js#L14

```js
const isAutoAddEnabled = configurationVault.getAutoAdd()
```

Reply:
Definitely. I did see how the way it works is addequate to solving the goals of the command, just that the error handling ends up being not QUITE right.

Just changing the command like these seemed the best way to just tweak the functionality to get to the goal.